nym-vpn-apple/NymVPNDaemon/Resources/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,61 +1,61 @@
 {
   "images" : [
     {
-      "filename" : "16-mac.png",
+      "filename" : "16.png",
       "idiom" : "mac",
       "scale" : "1x",
       "size" : "16x16"
     },
     {
-      "filename" : "32-mac 1.png",
+      "filename" : "32 1.png",
```

Comment:
Maybe we should rename these weirdly called PNG from `<size> 1.png` -> `<size>@2x.png` given that these are 16x16 with 2x factor?

Reply:
*More like a suggestion from me*